configs/peer/config.json

```diff
@@ -37,7 +42,7 @@
     "ACTOR_CHANNEL_CAPACITY": 100
   },
   "QUEUE": {
-    "MAXIMUM_TRANSACTIONS_IN_BLOCK": 8192,
+    "MAXIMUM_TRANSACTIONS_IN_BLOCK": 512,
```

Comment:
Great that now the file will be kept in sync. IIRC this queue change was merged quite some time ago.